python/MaterialX/main.py

```diff
@@ -292,3 +292,12 @@ def _stringToValue(string, t):
 #
 
 readFromXmlFile = readFromXmlFileBase
+
+
+#
+# Libraries
+#
+def getDefaultLibraryLocation():
```

Comment:
@kwokcb Going forward, I believe there's value in moving to the term "data libraries" instead of "libraries" when we're referring to the collection of MaterialX files that define the standard types, nodes, graphs, and targets.  What would you think about naming this helper function `getDefaultDataLibraryPath`, in order to facilitate this?

Reply:
This sounds good. I'll make the change.